src/Collections-Strings/Symbol.class.st

```diff
@@ -196,6 +196,12 @@ Symbol class >> rehash [
 	self resetSelectorTable
 ]
 
+{ #category : 'accessing' }
+Symbol class >> reservedLiterals [
+
+	^ PseudoVariable lookupDictionary keys
```

Comment:
This code only gives part of the reserved literals. Other needs to be hardcoded here on top of those (nil, true, false I think)